tests/neg/i1641.scala

```diff
@@ -2,7 +2,7 @@ package bar { object bippy extends (Double => String) { def apply(x: Double): St
 package object println { def bippy(x: Int, y: Int, z: Int) = "(Int, Int, Int)" }
 object Test {
   def main(args: Array[String]): Unit = {
-    println(bar.bippy(5.5))
+    println(bar.bippy(5.5)) // error
     println(bar.bippy(1, 2, 3)) // error
```

Comment:
Yes, but that's unrelated. Maybe they have the same position in dotty. Anyway we went from 1 to 2 errors, so it's an improvement :-)